index.html

```diff
@@ -154,7 +154,48 @@ <h3 class="skills-head"><img class="skill-bullets" src="./Work-section-img/squar
           </div>
         </div>
         </div>
+        </div>
+         
+        <div class="contact">
+            <div class="contact-grid">
+            <div class="contactpara">
+                <p class="contactp">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
```

Comment:
- [ ] Please, ensure that this text follows the template design guideline to achieve almost a 💯 pixel-perfect design 😉. Things like the line height, font size, weight et all. 😀 

- Your design
<img width="141" alt="wrong2" src="https://user-images.githubusercontent.com/41163571/220543575-6c2c2720-d758-4924-a314-964bab5f85e2.png">

- Figma design 🙃
<img width="226" alt="right2" src="https://user-images.githubusercontent.com/41163571/220543637-c43972bb-5106-4055-9b00-f89760b4d791.png">